src/test/scala/P14_Spec.scala

```diff
@@ -0,0 +1,25 @@
+import org.scalatest.FlatSpec
+
+class P14_Spec extends FlatSpec {
+
+  "P14_LongestCollatzSequence" should "calculate the correct result for statring number=14" in {
+    assert(9 === P14_LongestCollatzSequence.calculate(14))
+  }
+
+  "P14_LongestCollatzSequence" should "calculate the correct result for statring number=5847" in {
```

Comment:
replace repeating tests for the same object with ``it should ...``

Reply:
done with 271a13c